### ISSUE TITLE
Translation for the states of the mower

### ISSUE THREAD

**Issue by ulfur8149** [py-smart-gardena/hass-gardena-smart-system#40]
The gardena mower now has activity states like: "PARKED_PARK_SELECTED", "OK_LEAVING", ...
I was wondering is there a possiblity to work with a translation file and define these therms in English like "Parked until next scheduled mowing time", "Start leaving", ...
I'm willing to do then the dutch translation for the different options.

**Comment by zissou1** [comment]
I also request this, I could do the swedish translation.

**Comment by MarcBosma** [comment]
I also would like to see this implemented.
Here is a list i made with some standard states:
OK_SEARCHING = Terug naar dock
OK_CHARGING = Opladen
OK_LEAVING = Dock verlaten
OK_CUTTING = Maaien

![image](https://user-images.githubusercontent.com/52706067/120797184-f808ca00-c53b-11eb-810f-1f643769fa9d.png)

**Comment by grm** [comment]
Indeed, i am currently working on having a mower 'entity' in HASS, and hopefully it will include translations for all these states !

Jérémie

**Comment by grm** [comment]
Resolved in v2.x — the integration now uses the native Home Assistant `LawnMowerEntity` which provides translated states (mowing, docked, paused, error) handled by HA itself. Additionally, mower error codes are now translated (PR #319). The detailed Gardena activity values are exposed as extra state attributes. Closing as resolved.
